command/plugin_register.go

```diff
@@ -151,7 +156,7 @@ func (c *PluginRegisterCommand) Run(args []string) int {
 		c.UI.Error(fmt.Sprintf("Too many arguments (expected 1 or 2, got %d)", len(args)))
 		return 1
 	case c.flagSHA256 == "" && c.flagVersion == "":
-		c.UI.Error("One of -sha256 or -version is required. If registering with binary, please provide at least -sha256 (-version optional). If registering with extracted artifact directory, please provide -version only.")
+		c.UI.Error("One of -sha256 or -version is required. If registering with a binary, please provide at least -sha256 (-version optional). If registering with an artifact, please provide -version only.")
```

Comment:
I like how you broke up the line length in your changes above, mind doing the same here?